### PR TITLE
Small formatting adjustments to assignments

### PR DESCRIPTION
**Pull request by parker-codes**

### CODE REVIEW

File: assignments/week-0.md [hunk 1]
```diff
@@ -3,26 +3,32 @@
...
 	- `cd ~ && mkdir workspace`
 	- `cd workspace && mkdir hello-world`
+	- `cd hello-world`
 	- `git init`
 	- `git remote add origin {origin-from-your-github-repo}`
 	- `git add .`
 	- `git commit -m 'init commit'`
 	- `git push origin master`
+
+	(For reference, `cd` stands for `change directory` and `mkdir` stands for `make directory`. Just after creating the needed workspace and project directories, you are initializing git in that project and connecting it to the specific GitHub repository. The last steps of `add`, `commit`, and `push` are commands that you'll use often to update your changes with GitHub. Research any of these if they don't make sense!)
```
Comment (garrett-vangilder) [inline assignments/week-0.md:22]: 👍

File: assignments/week-0.md [hunk 2]
```diff
@@ -3,26 +3,32 @@
...
+---
 
-Once you have completed the code do the following
+Once you have completed the code do the following:
 
 1. Open a terminal
-1. `cd ~/workspace/hello-world`
-1. `git add .`
-1. `git commit -m 'finished the assignment'`
-1. `git push origin master`
```
Comment (garrett-vangilder) [inline assignments/week-0.md:28]: this was actually intentional - .md files will enumerate correctly for ordered lists as long as the integer is repeated. Kind of goofy but it does help with longer documents. Of course this is just you know my opinion haha

**Comment by garrett-vangilder** [comment]
Hey @google-mac remove `.DS_STORE` from your commit and then I'll approve and merge. Thank you for your hard work! Everything looks great!

File: assignments/week-0.md (hunk 2 above)
Comment (parker-codes) [reply assignments/week-0.md:28]: @garrett-vangilder I have some keybindings that resolve this for me, but I get what you mean!

**Comment by parker-codes** [comment]
@garrett-vangilder The file has been banished! (I wish gitignore could already have a list of those annoying system-control files..)

**Review by garrett-vangilder — APPROVED** [review]

**Pull request merged**